examples/custom_vocabularies_example.py

```diff
@@ -0,0 +1,52 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+from rev_ai import customvocabulariesclient
```

Comment:
@kbridbur What is python convention for naming clients? I'm afraid this client name is hard to read due to a lack of formatting.

snake case?

Reply:
I guess this isn't a client name, but rather a module name

Reply:
module names should be all lowercase, however it would be useful to insert some `_`s to make this more readable

Reply:
so snake_case I guess